core/node/rpc/archiver.go

```diff
@@ -101,6 +146,82 @@ func NewArchiver(
 	return a
 }
 
+func (a *Archiver) setupStatisticsMetrics(factory infra.MetricsFactory) {
```

Comment:
Emit internally tracked statistics as gauge metrics, a la what we do for postgres